expression/util.go

```diff
@@ -688,6 +688,19 @@ func pushNotAcrossExpr(ctx sessionctx.Context, expr Expression, not bool) (_ Exp
 	return expr, not
 }
 
+// GetExprInsideIsTruth get the expression inside the `istrue_with_null` and `istrue`.
+func GetExprInsideIsTruth(expr Expression) Expression {
```

Comment:
Besides, please add more comments to explain why we should do this.

Reply:
Updated.